src/components/Box/Box.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import classNames from 'classnames'
+import './Box.css'
+
+export type BoxProps = {
+  header?: string
+  className?: string
+  children: React.ReactNode
+  childrenClassName?: string
```

Comment:
is it necessary to have this children class name? what's the use case? couldn't it be themed using the other `className`? Like, if I wanted to customize this particular children's rules I could pass `className="my-custom-class"` and then `.my-custom-class .dcl.box-children { ... }`

Reply:
Yes, you are right. Thanks for the clarification. It has been removed